<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterpriseForm</class>
 <widget class="QWidget" name="EnterpriseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Copyright (c) DevSec Studio. All rights reserved.

MIT License

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the &quot;Software&quot;), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
*/

/*-----QWidget-----*/
QWidget
{
	background-color: #fff;
	color: red;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #454544;
	font-weight: bold;
	font-size: 13px;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #5c55e9;
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;

}


QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::hover
{
	background-color: #5564f2;

}


QPushButton::pressed
{
	background-color: #3d4ef2;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #5c55e9;
	font-size: 10px;
	font-weight: bold;
	border: none;
	border-radius: 5px;

}


/*-----QCheckBox-----*/
QCheckBox::indicator
{
    background-color: #323232;
    border: 1px solid darkgray;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;);
	background-color: #5c55e9;
    border: 1px solid #5c55e9;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #5c55e9;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #c2c7d5;
	color: #2a547f;
	border: none;
	padding: 5px;

}


/*-----QListView-----*/
QListView
{
	background-color: #5c55e9;
	color: #fff;
	font-size: 14px;
	font-weight: bold;
	show-decoration-selected: 0;
	border-radius: 4px;
	padding-left: -15px;
	padding-right: -15px;
	padding-top: 5px;

} 


QListView:disabled 
{
	background-color: #5c5c5c;

}


QListView::item
{
	background-color: #454e5e;
	border: none;
	padding: 10px;
	border-radius: 0px;
	padding-left : 10px;
	height: 32px;

}


QListView::item:selected
{
	color: #000;
	background-color: #fff;

}


QListView::item:!selected
{
	color:white;
	background-color: transparent;
	border: none;
	padding-left : 10px;

}


QListView::item:!selected:hover
{
	color: #fff;
	background-color: #5564f2;
	border: none;
	padding-left : 10px;

}


/*-----QTreeView-----*/
QTreeView 
{
	background-color: #fff;
	show-decoration-selected: 0;
	color: #454544;

}


QTreeView:disabled
{
	background-color: #242526;
	show-decoration-selected: 0;

}


QTreeView::item 
{
	border-top-color: transparent;
	border-bottom-color: transparent;

}


QTreeView::item:hover 
{
	background-color: #bcbdbb;
	color: #000;

}


QTreeView::item:selected 
{
	background-color: #5c55e9;
	color: #fff;

}


QTreeView::item:selected:active
{
	background-color: #5c55e9;
	color: #fff;

}


QTreeView::item:selected:disabled
{
	background-color: #525251;
	color: #656565;

}


QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings 
{
	image: url(://tree-closed.png);

}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  
{
	image: url(://tree-open.png);

}


/*-----QTableView &amp; QTableWidget-----*/
QTableView
{
    background-color: #fff;
	border: 1px solid gray;
    color: #454544;
    gridline-color: gray;
    outline : 0;

}


QTableView::disabled
{
    background-color: #242526;
    border: 1px solid #32414B;
    color: #656565;
    gridline-color: #656565;
    outline : 0;

}


QTableView::item:hover 
{
    background-color: #bcbdbb;
    color: #000;

}


QTableView::item:selected 
{
	background-color: #5c55e9;
    color: #fff;

}


QTableView::item:selected:disabled
{
    background-color: #1a1b1c;
    border: 2px solid #525251;
    color: #656565;

}


QTableCornerButton::section
{
	background-color: #ced5e3;
	border: none;
    color: #fff;

}


QHeaderView::section
{
	color: #2a547f;
	border: 0px;
	background-color: #ced5e3;
	padding: 5px;

}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    color: #fff;
    background-color: #5c55e9;

}


QHeaderView::section:checked:disabled
{
    color: #656565;
    background-color: #525251;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal 
{
    background-color: transparent;
    height: 8px;
    margin: 0px;
    padding: 0px;

}


QScrollBar::handle:horizontal 
{
    border: none;
	min-width: 100px;
    background-color: #7e92b7;

}


QScrollBar::add-line:horizontal, 
QScrollBar::sub-line:horizontal,
QScrollBar::add-page:horizontal, 
QScrollBar::sub-page:horizontal 
{
    width: 0px;
    background-color: #d8dce6;

}


QScrollBar:vertical 
{
    background-color: transparent;
    width: 8px;
    margin: 0;

}


QScrollBar::handle:vertical 
{
    border: none;
	min-height: 100px;
    background-color: #7e92b7;

}


QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical, 
QScrollBar::sub-page:vertical 
{
    height: 0px;
    background-color: #d8dce6;

}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="entrepriseEdit_Nom">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="entrepriseEdit_Nom_TB"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="entrepriseEdit_Ville">
           <property name="text">
            <string>Ville</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="entrepriseEdit_Ville_TB"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="entreprise_adresse">
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="entreprise_adresse_TB"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="entrepriseEdit_Tel">
         <property name="text">
          <string>Tel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="entrepriseEdit_Tel_TB"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="entrepriseEdit_Email">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="entrepriseEdit_Email_TB"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="entrepriseEdit_WebSite">
         <property name="text">
          <string>Site Web</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="entrepriseEdit_WebSite_TB"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="entrepriseEdit_RS">
         <property name="text">
          <string>RÃ©seaux socials</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QComboBox" name="entrepriseEdit_RS_CB"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="editRS">
         <property name="text">
          <string>Editer</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="deleteRS">
         <property name="text">
          <string>Suprimer</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="addRS">
         <property name="text">
          <string>Ajouter un nouveau reseaux</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QGridLayout" name="actionRS" rowminimumheight="0,0,0,0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item row="1" column="1">
        <widget class="QComboBox" name="listRS"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="confirmRS">
         <property name="text">
          <string>Confirmer Reseaux</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="entrepriseEdit_RsPseudo">
         <property name="text">
          <string>Identifiant :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLineEdit" name="entrepriseEdit_RsPseudo_TB"/>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="entrepriseDomaine">
             <property name="text">
              <string>Domaines</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QComboBox" name="listDomaineEntreprise"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="deleteDomaine">
             <property name="text">
              <string>Suprimer Domaine</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="listDomaines"/>
           </item>
           <item>
            <widget class="QPushButton" name="addDomaine">
             <property name="text">
              <string>Ajouter Domaine</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
